src/components/App/App.component.jsx

```diff
@@ -3,12 +3,14 @@ import { BrowserRouter, Switch, Route } from 'react-router-dom';
 
 import AuthProvider from '../../providers/Auth';
 import HomePage from '../../pages/Home';
-import LoginPage from '../../pages/Login';
+// import LoginPage from '../../pages/Login';
 import NotFound from '../../pages/NotFound';
-import SecretPage from '../../pages/Secret';
-import Private from '../Private';
-import Fortune from '../Fortune';
-import Layout from '../Layout';
+// import SecretPage from '../../pages/Secret';
+// import Private from '../Private';
+// import Fortune from '../Fortune';
+// import Layout from '../Layout';
```

Comment:
I suggest to remove the commented code, you always can check on git history the old version of the imports if you need it on the future